plugins/rbac-backend/src/service/enforcer-delegate.ts

```diff
@@ -179,15 +169,19 @@ export class EnforcerDelegate {
   ): Promise<void> {
     const trx = externalTrx ?? (await this.knex.transaction());
 
+    const policiesToAdd: string[][] = await policies.reduce<
+      Promise<string[][]>
+    >(async (accP: Promise<string[][]>, gp: string[]) => {
+      const acc = await accP;
+      const hasPolicy = await this.enforcer.hasGroupingPolicy(...gp);
```

Comment:
Same as the comment about `addPolicies`, we have multiple repeated checks of `hasGroupingPolicy` leading up to `addGroupingPolicies`. In `policies-rest-api.ts` we have the same call [here](https://github.com/janus-idp/backstage-plugins/blob/0d7ab0691878579c35b3d75a2cbd3bc1a5919f3b/plugins/rbac-backend/src/service/policies-rest-api.ts#L502) before `addGroupingPolicies`